bundles/org.openhab.automation.pwm/README.md

```diff
@@ -0,0 +1,62 @@
+# Pulse Width Modulation (PWM) Automation
+
+This automation module implements [Pulse Width Modulation (PWM)](https://en.wikipedia.org/wiki/Pulse-width_modulation).
+
+PWM can be used to control actuators continuously from 0 to 100% that only support ON/OFF commands. E.g. valves or heating burners.
+It accomplishes that by switching the actuator on and off with a fixed interval.
+The higher the control percentage (duty cycle), the longer the ON phase.
+
+Example: If the fixed interval is 10 sec and the duty cycle is 30%, the output is ON for 3 sec and OFF for 7 sec.
+
```

Comment:
I feel a warning is necessary about relay contact life expectancy. If this example was done for a full year it would be over 1.3 Million contact cycles which would wear most cheap devices out.